features/in_onderzoek.feature

```diff
@@ -356,83 +450,75 @@ Functionaliteit: in onderzoek
-      En heeft de persoon met burgerservicenummer '555550001' GEEN 'inOnderzoek' gegevens
-      En heeft de persoon met burgerservicenummer '555550001' GEEN 'geboorte.inOnderzoek' gegevens
+      | fields              | kinderen.geboorte               |
+      Dan heeft de response een persoon met een 'kind' met alleen de volgende 'geboorte' gegevens
+      | naam                                         | waarde       |
+      | plaats.code                                  | 0599         |
+      | inOnderzoek.datum                            | true         |
+      | inOnderzoek.plaats                           | true         |
+      | inOnderzoek.land                             | true         |
+      | inOnderzoek.datumIngangOnderzoek.datum       | 2022-03-07   |
+      | inOnderzoek.datumIngangOnderzoek.type        | Datum        |
+      | inOnderzoek.datumIngangOnderzoek.langFormaat | 7 maart 2022 |
 
     Abstract Scenario: partner heeft <nauwkeurigheid> soort verbintenis in onderzoek
       Gegeven het systeem heeft een persoon met de volgende gegevens
-      | naam                        | waarde    |
-      | burgerservicenummer         | 555550001 |
-      En de persoon heeft een partner met de volgende gegevens
       | naam                | waarde    |
-      | burgerservicenummer | 555550002 |
-      En de partner met burgerservicenummer '555550002' heeft volgende 'inOnderzoek' gegevens
-      | naam                          | waarde   |
-      | aanduidingGegevensInOnderzoek | <waarde> |
-      | datumIngangOnderzoek          | 20220307 |
+      | burgerservicenummer | 555550001 |
+      En de persoon heeft een 'partner' met de volgende gegevens
+      | naam                                     | waarde   |
+      | aanduiding gegevens in onderzoek (83.10) | <waarde> |
+      | datum ingang onderzoek (83.20)           | 20220307 |
       Als personen wordt gezocht met de volgende parameters
       | naam                | waarde                          |
       | type                | RaadpleegMetBurgerservicenummer |
       | burgerservicenummer | 555550001                       |
       | fields              | partners                        |
-      Dan heeft de partner met burgerservicenummer '555550002' alleen de volgende 'inOnderzoek' gegevens
-      | naam                             | waarde     |
-      | soortVerbintenis                 | true       |
-      | datumIngangOnderzoek.datum       | 2022-03-07 |
-      | datumIngangOnderzoek.type        | Datum      |
-      | datumIngangOnderzoek.langFormaat | 7 maart 2022 |
-      En heeft de partner met burgerservicenummer '555550002' GEEN 'naam.inOnderzoek' gegevens
-      En heeft de partner met burgerservicenummer '555550002' GEEN 'geboorte.inOnderzoek' gegevens
-      En heeft de persoon met burgerservicenummer '555550001' GEEN 'persoonInOnderzoek' gegevens
+      Dan heeft de response een persoon met een 'partner' met alleen de volgende gegevens
```

Comment:
Dit geldt toch alleen maar als regel 507 gelezen moet worden als 

'En de persoon heeft een 'partner' met **alleen** de volgende gegevens'

Ik zou dat expliciet maken omdat je anders de conclusie kunt trekken dat ook als er nog andere gegevens zijn je toch onderstaande response moet geven.

Reply:
ik vind het prima om het toe te voegen, maar ik wil nogmaals benadrukken dat ik het heb overgenomen zoals het initieel staat. Daar stond het ook niet. Ik heb in de p.r. alleen de scenario's aangepast zodat ze kunnen worden gebruikt als test en niet gekeken of het tekstueel klopt.